docs/en/observability/uptime-set-up-heartbeat.asciidoc

```diff
@@ -0,0 +1,130 @@
+[[uptime-set-up-choose-heartbeat]]
+= Use {heartbeat}
+
+{heartbeat} is a lightweight daemon that you install on a remote server to periodically
+check the status of your services and determine if they are available. Performance data is
+then gathered and reported back to {heartbeat}, where it's formatted and sent to the {stack}.
+
+[discrete]
+[[uptime-set-up-docker]]
+== Pull the Docker image
+
+Elastic provides Docker images that you can use to run monitors, which reduces the chance of problems.
+Start by pulling the {heartbeat} Docker image.
+
+// ifeval::["{release-state}"=="unreleased"]
+
+// Version {version} has not yet been released.
+
+// endif::[]
+
+// ifeval::["{release-state}"!="unreleased"]
+
+[source,sh,subs="attributes"]
+----
+docker pull docker.elastic.co/beats/heartbeat:{version}
+----
+
+// endif::[]
+
+[discrete]
+[[uptime-set-up-config]]
+== Configure
+
+Next, create a `heartbeat.yml` configuration file.
+
+There are four possible monitor types.
+The example below shows how to configure an `http` monitor.
+
+[source,yml]
+----
+heartbeat.monitors:
+- type: http
+  id: service-status <1>
+  name: Service Status
+  service.name: my-apm-service-name
+  hosts: ["http://localhost:80/service/status"]
+  check.response.status: [200]
+  schedule: '@every 5s'
+----
+<1> Each `monitor` gets its own ID in the {uptime-app} and its own schedule entry.
+This allows tests to be run in parallel and analyzed separately.
+
+Read more about configuration options in {heartbeat-ref}/configuration-heartbeat-options.html[Configure {heartbeat} monitors].
+
+If you are setting up a **`browser`** monitor, see <<synthetics-create-test>>
+for more details on writing synthetic tests to be added to the configuration file.
```

Comment:
This phrase sounds confusing, especially the last part about "for more details on writing synthetic tests to be added to the configuration file".

I don't understand what this means, so can't provide suggestions, but does sound a bit weird IMO 😢

Reply:
Is this clearer? 

```suggestion
NOTE: If you are setting up a **`browser`** monitor, see <<synthetics-create-test>>
for more details on writing synthetic tests. You can add tests directly to the configuration file 
using https://www.elastic.co/guide/en/beats/heartbeat/current/monitor-browser-options.html#monitor-source-inline[`inline`] or reference tests stored elsewhere using https://www.elastic.co/guide/en/beats/heartbeat/current/monitor-browser-options.html#monitor-source-zipurl[`zip_url`].
```

Reply:
I like the rewording, but I suppose I didn't find the original confusing myself.

Reply:
I like the rewording, just one nit:

Instead of:

```
You can add tests directly to the configuration file [...]
```

Maybe we should say:

```
You can add tests directly when configuring the monitor through the UI [...] 
```